modules/ROOT/pages/tls-configuration.adoc

```diff
@@ -21,6 +21,9 @@ The TLS Configuration element is independent of any module or transport. In Mule
 
 You can create this element through the UI of the HTTP Connector's Global configuration element, on the *TLS/SSL* tab, or through the UI of the Web Service Consumer, on the *Security* tab.
 
+[NOTE]
+Other connectors may support TLS. If they do, you'll see a *TLS/SSL* tab on that connector's Global configuration element.
```

Comment:
"you'll" is unclear wording. How about rewording these 2 sentences as: "If a Mule 3 connector supports TLS, there's a TLS/SSL tab in the connector's Global Element Properties."

Reply:
will resolve as noted in v3.8.